### PR TITLE
Feature/fx cop fixes

### PR DESCRIPTION
**Pull request by AlexandreSanscartier**
I added [StyleCop](https://documentation.help/StyleCop/StyleCop%20Rules.html) to the project it basically checks the code every time we build to make sure we're following the right coding conventions. Right now we have the default rules for StyleCop, but we can decide as a team if ever we want to suppress some globally or in certain situations (**Not this should be used sparingly**). 

I also cleaned up some of the code based on conventions I've used in the past and for readability/clarity. I'm also looking for your comments in particular. In a few places I've added the comment // Perhaps we can fail more gracefully then just throwing an exception I'm wondering if we can have a discussion on how best to proceed. 

Thanks!

### CODE REVIEW

**Comment by AlexandreSanscartier** [comment]
I'll add the action in particular to the Repository once this PR is closed. It's called .NET Core and we can configure it to build and check our builds for any errors.

File: CDTS Blazor/Data/Services/AzureBlobService.cs [hunk 1]
```diff
@@ -20,41 +22,45 @@ public AzureBlobService(IAzureBlobConnectionFactory azureBlobConnectionFactory)
...
             }
 
             return blob;
-
         }
 
         public async Task<List<CloudBlockBlob>> UploadMultipleFilesAsync(IFormFileCollection files, string container = null)
-
         {
+            // Perhaps we can fail more gracefully then just throwing an exception
```
Comment (TCPrimedPaul) [inline CDTS Blazor/Data/Services/AzureBlobService.cs:46]: Wouldn't that be up to the UI?

**Review by TCPrimedPaul — COMMENTED** [review]
I had a look out of curiosity but also because I am new to GitHub and need to start somewhere ;)

File: CDTS Blazor/Data/Services/AzureBlobService.cs (hunk 1 above)
Comment (AlexandreSanscartier) [reply CDTS Blazor/Data/Services/AzureBlobService.cs:46]: That's a good point we definitely need a way of surfacing those up to the UI in some way.

**Review by tc-tibo — APPROVED** [review]
Great stuff. Makes sense to rename the project and the error handling class is a good practice.

**Comment by tc-tibo** [comment]
> 
> 
> I added [StyleCop](https://documentation.help/StyleCop/StyleCop%20Rules.html) to the project it basically checks the code every time we build to make sure we're following the right coding conventions. Right now we have the default rules for StyleCop, but we can decide as a team if ever we want to suppress some globally or in certain situations (**Not this should be used sparingly**).
> 
> I also cleaned up some of the code based on conventions I've used in the past and for readability/clarity. I'm also looking for your comments in particular. In a few places I've added the comment // Perhaps we can fail more gracefully then just throwing an exception I'm wondering if we can have a discussion on how best to proceed.
> 
> Thanks!

For comments that require a follow-up, perhaps we could start using tokens to make use of Visual Studio's Task List.

https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2019

E.g.

`// TODO: Perhaps we can fail more gracefully then just throwing an exception`

**Pull request merged**